requirements.txt

```
@@ -3,4 +3,5 @@ future
 netaddr
 psutil
 rfc5424-logging-handler
+wmi
```

```suggestion
wmi ; sys_platform == 'win32'
```
Sorry that I did not catch this the first time but README.md says we only want this dependency on Windows.